### ISSUE TITLE
Feature request: file blacklisting

### ISSUE THREAD

**Issue by alzhur** [Kalanyr/gogrepoc#49]
Main reason for me - no language attribute for extras, no good way to skip downloading extras for all languages.
Proposed solution:
1. Add command to add file(s) to blacklist, added files not count as missing on running `gogrepoc.py verify` etc. Count as orphaned, maybe.
2. `gogrepoc.py download -skipfiles` skips blacklisted files if no filenames specified directly, or add another option, or make blacklisted files skipped by default.

**Comment by alzhur** [comment]
Any chances to implement this?

**Comment by Kalanyr** [comment]
Yeah, I'm just looking after a sick relative at the moment and their
internet is not good enough for this kinda thing so stuff is going to be on
hold for a while except for actual breakages.
